doc/src/modules/polys/ringseries.rst

```diff
@@ -149,7 +149,7 @@ polynomials (see :class:`sympy.polys.polyoptions.Options`). Thus, when we want
 ``sring`` to allow rational exponents we supply a ``series=True`` flag to
 ``sring``::
 
-    >>> rs_series(sin(a**QQ(1, 2)), a, 3) # doctest: +SKIP
+    >>> rs_series(sin(a**QQ(1, 3)), a, 3) # doctest: +SKIP
     -1/5040*a**(7/3) + 1/120*a**(5/3) - 1/6*a + a**(1/3)
```

Comment:
Why is the doctest skipped? Is it because it previously gave the wrong result?

Reply:
It was skipped when added. I guess it was because the contributor added this didn't know why it couldn't pass and skipping it is the most easiest solution.
I also remove another +SKIP and fix the error.